src/main/scala/intellij/haskell/psi/HaskellLayoutLexer.scala

```diff
@@ -36,17 +36,18 @@ class HaskellLayoutLexer(private val lexer: Lexer,
     override def toString = s"${elementType.toString} ($start, $end)"
 
     val isEOF: Boolean = elementType.isEmpty
-    val isCode: Boolean = elementType.exists(et => !nonCodeTokens.contains(et)) && !isEOF
+    def isCode: Boolean = elementType.exists(et => !nonCodeTokens.contains(et)) && !isEOF
```

Comment:
`isCode` can be `val` because depends on `val`s

Reply:
Yes. This was a mistake.